spec/intl.html

```diff
@@ -11,6 +11,68 @@ <h1>Amendments to the ECMAScript® 2021 Internationalization API Specification</
+    </p>
+
+    <emu-clause id="sec-intl-isvalidtimezonename">
+      <h1><a href="https://tc39.es/ecma402/#sec-isvalidtimezonename">IsValidTimeZoneName</a> ( _timeZone_ )</h1>
+
+      <p>
+        <del>The</del> IsValidTimeZoneName <ins>is a host-defined</ins> abstract operation <ins>which</ins> verifies that the _timeZone_ argument (which must be a String value) represents a valid Zone or Link name of the IANA Time Zone Database.
+        <ins>A conforming implementation must implement the abstract operation as follows:</ins>
+      </p>
+      <p>
+        The abstract operation returns true if _timeZone_, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>, is equal to one of the Zone or Link names of the IANA Time Zone Database, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>. It returns false otherwise.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-intl-canonicalizetimezonename">
+      <h1><a href="https://tc39.es/ecma402/#sec-canonicalizetimezonename">CanonicalizeTimeZoneName</a> <ins>( _timeZone_ )</ins></h1>
+
+      <p>
+        <del>The</del> CanonicalizeTimeZoneName <ins>is a host-defined</ins> abstract operation <ins>which</ins> returns the canonical and case-regularized form of the _timeZone_ argument (which must be a String value that is a valid time zone name as verified by the IsValidTimeZoneName abstract operation).
+        <del>The following steps are taken:</del>
+        <ins>A conforming implementation must implement the abstract operation as follows:</ins>
+      </p>
+
+      <emu-alg>
+        1. Let _ianaTimeZone_ be the Zone or Link name of the IANA Time Zone Database such that _timeZone_, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>, is equal to _ianaTimeZone_, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>.
+        1. If _ianaTimeZone_ is a Link name, let _ianaTimeZone_ be the corresponding Zone name as specified in the *"backward"* file of the IANA Time Zone Database.
+        1. If _ianaTimeZone_ is *"Etc/UTC"* or *"Etc/GMT"*, return *"UTC"*.
+        1. Return _ianaTimeZone_.
+      </emu-alg>
+
+      <p>
+        The Intl.DateTimeFormat constructor allows this time zone name; if the time zone is not specified, the host environment's current time zone is used. Implementations shall support UTC and the host environment's current time zone (if different from UTC) in formatting.
```

Comment:
As far as I can tell, we can't require supporting the host environment's current time zone (`Camlandia/Camville`), I think the requirement that they support it if it's returned by DefaultTimeZone would fix this

Reply:
This is actually the already-existing text in ECMA-402 (the Temporal changes in this section are marked with `<ins>` and `<del>`) but I agree it would be good to mention DefaultTimeZone here.